Examples/Algorithms/Fatras/include/ActsExamples/Fatras/FatrasAlgorithm.hpp

```diff
@@ -35,6 +36,8 @@ class FatrasAlgorithm final : public BareAlgorithm {
     std::string outputParticlesFinal;
     /// The simulated hits output collection.
     std::string outputSimHits;
+    /// Parametrisation of nuclear interaction
+    std::string imputParametrisation = "parameters";
```

Comment:
Can this be a bit more specified - what's parameters?

Reply:
Renamed